tutorials/noisepy_scedc_tutorial.ipynb

```diff
@@ -272,10 +275,21 @@
       "outputs": [],
       "source": [
         "#stations = \"RPV,STS,LTP,LGB,WLT,CPP,PDU,CLT,SVD,BBR\".split(\",\") # filter to these stations\n",
-        "stations = \"RPV,SVD,BBR\".split(\",\") # filter to these stations\n",
+        "# stations = \"RPV,SVD,BBR\".split(\",\") # filter to these stations\n",
```

Comment:
I would leave this as the default mode and the file loading below as a commented out example